### DISCUSSION TITLE
Do you have a documentation?

### DISCUSSION THREAD

**Discussion by azemerov** [paolo-rossi/delphi-neon#61] [General]
Do you have a documentation? The README.md contains only basic information.

**Comment by paolo-rossi** [comment]
Hi @azemerov ,

I'm afraid not much!  Please have a look at the Demos, they cover several use cases, plus I wrote an article about Neon on my blog: https://blog.paolorossi.net/ 

And, of course, if you need anything please reach me trough this Neon forum

Ciao,
Paolo.
